src/indexer.c

```diff
@@ -15,6 +15,17 @@ RedisModuleDict *labelsIndex;
 #define KV_PREFIX "__index_%s=%s"
 #define K_PREFIX "__key_index_%s"
 
+void _deepCopyDict(RedisModuleCtx *ctx, RedisModuleDict *dest, RedisModuleDict *src) {
+    /*
+     * Copy all elements from src to dest
```

Comment:
might be good to add to the comment / indicate in the function name that the function only copies the keys and not the values

Reply:
Given Danni's comment, the name is changed to _union, so it works the same as the _intersection etc.